src/patternfly/components/Card/examples/Card.md

```diff
@@ -191,10 +191,29 @@ cssPrefix: pf-c-card
 {{/card}}
 ```
 
+```hbs title=Selectable
```

Comment:
What do you think about updating this to be `.pf-m-selectable` (remove `.pf-m-selected`) and adding another example for "selected" with `.pf-m-selectable.pf-m-selected`?

Reply:
wfm